File: python/7 kyu/412.py
# https://www.codewars.com/kata/58880c6e79a0a3e459000004/train/python

# Passed

def house_numbers_sum(inp):
    result = 0
    for num in inp:
        if num == 0:
            break
            
        result += num
        
    return result

output = house_numbers_sum([5, 1, 2, 3, 0, 1, 5, 0, 2])
print(output)

File: nodejs/7 kyu/412.js
// https://www.codewars.com/kata/5c556845d7e0334c74698706/train/javascript

// Passed

function fit_in(a, b, m, n) {
  let squareSideSum = a + b;
  let biggerSide = Math.max(a, b);
  let result =
    (squareSideSum <= m && biggerSide <= n) ||
    (squareSideSum <= n && biggerSide <= m);
  return result;
}

const output = fit_in(1, 2, 3, 2);
console.log(output);